doc/CaliptraTrademarkAudit.md

```diff
@@ -0,0 +1,72 @@
+
+# Caliptra Trademark Auditing Process
+
+This document will describe the specific requirements to be reviewed by a 3rd party and attest to Caliptra TAC that the requirements have been met.
+
+The goal of a Caliptra Trademark Audit would be to be compliant with OCP Safe Level 2 requirements using an OCP Safe approved auditor.
+
+Currently, this is a list of Topics to review for Trademark Compliance.  Specific testable requirements will need to be spelled out subsequently.
+
+ * Each of these topics will have to be reviewed for an integration to get approval of Trademark Usage
+
+## Caliptra IP
+
+* Show how the RTL was integrated into the device environment
+* Show a Run of the tool to verify the RTL Release Hash (in development)
+* Show the ROM hash from a release ROM hash
+* Show the internal Caliptra FW signing process used to sign Caliptra FW
+* Show what options were used to synthesize the RTL
+* Show input wires to RTL
```

Comment:
We have an integration guide. Can we reference the sections in said guide?

Same for options used to synthesize